inspectit-oce-documentation/src/docs/asciidoc/_includes/metrics/metrics-exporters.adoc

```diff
@@ -0,0 +1,43 @@
+=== Metrics Exporters
+
+Metrics exporters are responsible for passing the collected metrics to metric storage.
+They can implement a push approach where metrics are sent to a collector or a pull approach where metrics are scraped by an external system.
+
+If an exporter supports run-time updates it means that it can be enabled/disabled during the run-time or that any property related to the exporter can be changed.
+This way you can, for example, change the endpoint where exporter pushes the metrics without a need to restart the application.
+In order to use run-time updates, you must enable one of the <<Externalized Configuration Methods,externalized configuration methods>> that support dynamic updates.
+
+inspectIT OCE current supports following OpenCensus metrics exporters:
+
+[cols="1,1,1,1",options="header"]
+|===
+|Exporter |Supports run-time updates| Push / Pull |Enabled by default
+|<<Prometheus Exporter>>
+|Yes
+|Pull
+|Yes
+|===
+
+==== Prometheus Exporter
+
+Prometheus exporter exposes the metrics in Prometheus format and is the default metrics exporter set up by inspectIT OCE.
+It requires that a Prometheus HTTP server is started in parallel with your application.
```

Comment:
This sentence soudns a bit like you have to manualyl start something imo.
More like "When enabled, inspectIT starts a Prometheus server. The server is by default started on port....